services/worker/src/worker/main.py

```diff
@@ -32,9 +39,15 @@ async def every(
         await asyncio.sleep(seconds)
 
 
+class JobCrashedError(CustomError):
+    def __init__(self, message: str, cause: Optional[BaseException] = None):
+        super().__init__(message, HTTPStatus.NOT_IMPLEMENTED, "JobCrashedError", cause, True)
```

Comment:
Not sure if we should disclose the error to the user here - maybe I should set it to False
```suggestion
        super().__init__(message, HTTPStatus.NOT_IMPLEMENTED, "JobCrashedError", cause, False)
```